README.md

```diff
@@ -68,70 +104,110 @@ Below is an example of a [settings.json](https://code.visualstudio.com/Docs/cust
+* Use a non-blinking block cursor
+* Type: Boolean (Default: `false`)
+
+#### `"vim.useCtrlKeys"`
+* Enable Vim ctrl keys overriding common VS Code operations (eg. copy, paste, find, etc). Enabling this setting will:
     * `ctrl+c`, `ctrl+[` => `<Esc>`
     * `ctrl+f` => Full Page Forward
     * `ctrl+d` => Half Page Back
     * `ctrl+b` => Half Page Forward
     * `ctrl+v` => Visual Block Mode
     * etc.
-  * Type: Boolean (Default: `true`)
-  * *Example:*
-
-    ```
-    "vim.useCtrlKeys": true
-    ```
+* Type: Boolean (Default: `true`)
 
-#### handleKeys
-  * Allows user to select certain modifier keybindings and delegate them back to VSCode so that VSCodeVim does not process them.
-  * Complete list of keys that can be delegated back to VSCode can be found in our [package.json](https://github.com/VSCodeVim/Vim/blob/master/package.json#L44). Each key that has a vim.use<C-...> in the when argument can be delegated back to vscode by doing "<C-...>":false.
-  * An example would be if a user wanted to continue to use ctrl + f for find, but wants to have useCtrlKeys set to true so that other vim bindings work.
+#### `"vim.handleKeys"`
+* Allows user to select certain modifier keybindings and delegate them back to VSCode so that VSCodeVim does not process them.
+* Complete list of keys that can be delegated back to VSCode can be found in our [package.json](https://github.com/VSCodeVim/Vim/blob/master/package.json#L44). Each key that has a vim.use<C-...> in the when argument can be delegated back to vscode by doing "<C-...>":false.
+* An example would be if a user wanted to continue to use ctrl + f for find, but wants to have [`useCtrlKeys`](#vimusectrlkeys) set to true so that other vim bindings work.
 
-    ```
-    "vim.handleKeys":{
+```json
+    "vim.handleKeys": {
         "<C-a>": false,
         "<C-f>": false
     }
-    ```
+```
+
+### Key remapping
 
-#### insertModeKeyBindings/otherModesKeyBindings
-  * Keybinding overrides to use for insert and other (non-insert) modes.
+There's several different settings you can use to define custom remappings. Also related are the [`useCtrlKeys`](#vimusectrlkeys) and [`handleKeys`](#vimhandlekeys) settings.
+
+#### `"vim.insertModeKeyBindings"`/`"vim.otherModesKeyBindings"`
+* Keybinding overrides to use for insert and other (non-insert) modes.
 
 Bind `jj` to `<Esc>` in insert mode:
 
+```json
+    "vim.insertModeKeyBindings": [
+        {
+            "before": ["j", "j"],
+            "after": ["<Esc>"]
+        }
+    ]
 ```
-  "vim.insertModeKeyBindings": [
-       {
-           "before": ["j", "j"],
-           "after": ["<Esc>"]
-       }
-  ]
-```
+
 Bind `:` to show the command palette:
 
-```
+```json
 "vim.otherModesKeyBindingsNonRecursive": [
-   {
-       "before": [":"],
-       "after": [],
-       "commands": [
-           {
-               "command": "workbench.action.showCommands",
-               "args": []
-           }
-       ]
-   }
+    {
+        "before": [":"],
```

Comment:
The indentation was all whacky, I re-indented it all so it was consistent. (I think there was 3-spaces here)